lambdas/cloudfront-security/replicator.js

```diff
@@ -52,18 +52,19 @@ exports.handler = async (event, context) => {
       if (event.RequestType === 'Create') {
         console.log('Creating function')
         const createResult = await lambdaEdge.createFunction({
-          Code: {
-            ZipFile: zipped
-          },
-          Description: '',
-          FunctionName: functionName,
-          Handler: 'index.handler',
-          MemorySize: 128,
-          Role: event.ResourceProperties.RoleArn,
-          Runtime: 'nodejs12.x',
-          Timeout: 1
+            Code: {
+              ZipFile: zipped
+            },
+            Description: '',
+            FunctionName: functionName,
+            Handler: 'index.handler',
+            MemorySize: 128,
+            Role: event.ResourceProperties.RoleArn,
+            Runtime: 'nodejs12.x',
+            Timeout: 1
```

Comment:
Please revert this. It's just diff noise and it already previously matched the surrounding code.
```suggestion
          Code: {
            ZipFile: zipped
          },
          Description: '',
          FunctionName: functionName,
          Handler: 'index.handler',
          MemorySize: 128,
          Role: event.ResourceProperties.RoleArn,
          Runtime: 'nodejs12.x',
          Timeout: 1
```